### PR TITLE
Fix backup helm charts

### PR DESCRIPTION
**Pull request by vvaltia**
Signed-off-by: Valtteri Valtia <vvaltia@csc.fi>

### CODE REVIEW

File: sda-svc/templates/backup-deploy.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.backup.deploy}}
+{{- if .Values.global.backup.deploy}}
```
Comment (blankdots) [inline sda-svc/templates/backup-deploy.yaml:1]: i think it is without the `global`
```suggestion
{{- if .Values.backup.deploy}}
```

File: sda-svc/templates/backup-secrets.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.backup.deploy}}
+{{- if .Values.global.backup.deploy}}
```
Comment (blankdots) [inline sda-svc/templates/backup-secrets.yaml:1]:
```suggestion
{{- if .Values.backup.deploy}}
```

**Review by blankdots — APPROVED** [review]
see line https://github.com/CSCfi/sda-helm/blob/master/sda-svc/values.yaml#L313 i don't think `.global` is needed.

**Comment by vvaltia** [comment]
Oh. Was a bit confused with these variablres

**Review by blankdots — APPROVED** [review]

**Pull request merged**